src/libcrun/cgroup.c

```diff
@@ -914,8 +914,9 @@ libcrun_cgroup_enter (oci_container_linux_resources *resources, int cgroup_mode,
       return ret;
     }
 
-  if (rootless > 0 && cgroup_mode != CGROUP_MODE_UNIFIED)
+  if (rootless > 0 && (cgroup_mode != CGROUP_MODE_UNIFIED || manager != CGROUP_MANAGER_SYSTEMD))
```

Comment:
Should we at least log a debug message if manager != SYSTEMD?

Reply:
I've added a warning when `manager != SYSTEMD` and running on cgroups v2